### PR TITLE
[16.0][MIG]+[IMP] module_change_auto_install : port from 15.0 to 16.0 add advanced configuration

### PR DESCRIPTION
**Pull request by legalsylvain**
- Port of the module ``module_change_auto_install``. (see original PR : https://github.com/OCA/server-tools/pull/2091)

- Add extra possibilities to use the recent syntax that allow ``auto_install`` value to be a list of modules and not just a boolean.

Please review only last commit.

CC : @kafai-lam, @extrememicro, @moylop260

### CODE REVIEW

**Review by rvalyi — APPROVED** [review]

File: module_change_auto_install/patch.py [hunk 1]
```diff
@@ -8,56 +8,84 @@
...
 
 def post_load():
-    _logger.info("Applying patch module_change_auto_intall")
-    modules.module.load_information_from_description_file = (
-        _overload_load_information_from_description_file
-    )
-    modules.load_information_from_description_file = (
-        _overload_load_information_from_description_file
-    )
+    _logger.info("Applying patch module_change_auto_intall ...")
```
Comment (thomaspaulb) [inline module_change_auto_install/patch.py]: Small typo

**Review by thomaspaulb — CHANGES_REQUESTED** [review]

File: module_change_auto_install/patch.py (hunk 1 above)
Comment (legalsylvain) [reply module_change_auto_install/patch.py]: Done. Thanks for your review. coud you update your review ?

**Review by thomaspaulb — APPROVED** [review]

**Comment by legalsylvain** [comment]
@sbidoul : any idea regarding the last bot message ? 

AFAIK : 
- understanding order of merge : OK
- wait for CI : OK
- merge : OK
- upload to pypi : failed.

**Comment by sbidoul** [comment]
Looks like a transient error on PyPI. Strange that the merge was successful, though.

**Pull request merged**